qcodes/instrument_drivers/Keithley/Keithley_2600_channels.py

```diff
@@ -0,0 +1,968 @@
+import logging
```

Comment:
since this is a new file, and the old driver has been removed already, we can remove the _channels suffix, i think :)